### ISSUE TITLE
feat(developer/ide): keep font/size consistent, maybe +/- controls

### ISSUE THREAD

**Issue by srl295** [keymanapp/keyman#10711]
### Is your feature request related to a problem? Please describe.

Can't see and had the editor on a big screen. Changed the 'Character Font' but the 'Code Font' was not changed at the same time, giving an odd effect.

### Describe the solution you'd like

- a hotkey to increase/dcrease the font wsize would be the nicest actually - control+ and control-
- perhaps keep the font _size_ in sync between the code font and character font? 

### Describe alternatives you've considered

- doing nothing
- not filing this report

### Related issues

#10456 

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [ ] Keyman for Windows
- [X] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

17.0.267-alpha

### Operating system

windows 11

**Comment by mcdurdin** [comment]
Yes this is an itch I want to scratch. The font size needs to be able to be decoupled because some scripts need larger fonts to be easily readable alongside Latin (front of mind is Thai). But for many it's better for the sizes to be linked for sure.

**Comment by mcdurdin** [comment]
Related, from FV team (@caforbes, @HopsAndHops, @xyzhelen) feedback:

> It would be nice to change the default font size for the code editor globally, instead of having to do it every time we open a different project.
